src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
There is 53 flow errors in `ReactCompositeComponent` (missing annotations), I'll do this file in a separate PR.